src/main/kotlin/org/wycliffeassociates/otter/common/domain/ProjectPageActions.kt

```diff
@@ -48,17 +56,41 @@ class ProjectPageActions(
     }
 
     fun createNewTake(chunk: Chunk, project: Collection, chapter: Collection): Single<Take> {
-        return getTakeCount(chunk)
-                .map { numOfTakes ->
+        return getHighestTakeNumber(chunk)
+                .map { highestNumber ->
                     // Create a file for this take
+                    val languageSlug = project.resourceContainer?.language?.slug ?: ""
+                    val rcSlug = project.resourceContainer?.identifier ?: ""
+                    val bookNumber = "%02d".format(
+                            // Handle book number offset
+                            if (project.sort > 39) project.sort + 1 else project.sort
```

Comment:
the resource container does kind of let us down here. (Actually, arguably UFW does, I don't remember why Matthew starts at 41, but I see no reason why they didn't make the sort the book number here. It still sorts the same.)

This branch will make the book number of a file name ALWAYS skip 40, regardless of whether it's of the Bible or not. We'd probably want to just check if it's a bible project first.